tests/probes/test_probes_packagehallucination.py

```diff
@@ -18,3 +19,6 @@ def test_promptcount():
     assert (
         len(p_ruby.prompts) == expected_count
     ), f"Ruby prompt count mismatch. Expected {expected_count}, got {len(p_ruby.prompts)}"
+    assert (
+        len(p_javascript.prompts) == expected_count
+    ), f"JavaScript prompt count mismatch. Expected {expected_count}, got {len(p_javascript.prompts)}"
```

Comment:
starting to feel a loop is possible here..

Reply:
Updated